docs/backend/README.md

```diff
@@ -154,3 +154,20 @@ To update python dependencies, update the [`Pipfile`](../../Pipfile) and run `ma
 You will also need to rebuild local running docker containers by running `make local-rebuild`.
 
 If you add a third-party library (directly or indirectly) that does not support [python typing](https://docs.python.org/3/library/typing.html), then you may need to add an entry to [`mypy.ini`](../../mypy.ini) to let mypy know [not to expect type hints for that library](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-type-hints-for-third-party-library).
+
+
+## Adding new users to the staging db:
+* get username / password credentials from aws secrets manager (:
+  * `aws secretsmanager get-secret-value --secret-id aspen-config`
+  
+* get RDS host and port information:
+  * `aws rds describe-db-instances --db-instance-identifier aspen-db`
+  
+* connect to aspen db using psql:
+  * `psql -h <RDS host> -p <RDS port> -U user_rw aspen_db`
+    ```sql
+    aspen_db=> set search_path to aspen;
+    aspen_db=> INSERT INTO users (name, email, auth0_user_id, group_admin, system_admin, group_id) VALUES ('<name>', '<email>', '<auth0 user ID>', 'f', 't', <group ID>);
+    ```
+    * to see all possible group ids:
+      * `select * from groups;`
```

Comment:
also note that this procedure is volatile for the time being as it can be wiped when we wipe the staging DB.

and doing this from `aspencli db interact` may be easier.

Reply:
that's true, i was just adding what i did to add the users and documenting at the same time, should polish this up later..